### DISCUSSION TITLE
Pseudoklassen vs Styled Components vs Github Linter

### DISCUSSION THREAD

**Discussion by devchristiansimon** [neuefische/questions#48] [Web] [answered]
Ich bin gerade auf ein sehr seltsames Problem gestoßen.
Und zwar nutze ich ja styled components. Dort wollte ich für einen Button einen Hover-Effekt machen

`&:hover {
background-color: #db0000;
}`

(Der Code ist auf 3 Zeilen aufgebaut. Klappt hier irgendwie nicht).

Soweit so gut. Als ich das pushen wollte und hier im Github die Lintermaschine gerödelt hat, hat er mir einen Fehler ausgespuckt.
In meinem Pullrequest könnt ihr das lesen: https://github.com/devchristiansimon/capstone-project/pull/10

Offenbar will der Linter, dass dort eine zusätzliche leere Zeile zwischen den geschweiften Klammern ist. Wenn ich die aber reinmache und speichere, ändert prettier das sofort wieder um. 

Hilfe ^^

**Comment by pixelass** [comment, marked as answer]
Das scheint aber zu klappen:

<img width="1057" alt="Screen Shot 2022-08-03 at 23 01 23" src="https://user-images.githubusercontent.com/1148334/182711019-c170f63a-ca35-4d18-9706-c8365ce73b3c.png">


BTW: code blocks kannst du mit 3 ` machen


````
```css
&:hover {
  background: red;
}
```
````

```css
&:hover {
  background: red;
}
```
